fecgov/fec-cms
# 5780 - Add 2012, 2008 to the presidential finance map

## Summary

- Resolves #5780 

Adding 2008 and 2012 to the presidential finance map

### Required reviewers

- UX?
- SMEs

## Impacted areas of the application

Only the map (after back-end/db confirm performance is acceptable)

## Screenshots

<img width="972" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/762fecef-32cb-4cb9-833a-1ef6ffd46109">

## Related PRs

None

## How to test
- pull the branch
- `npm run build`
- `./manage.py runserver`
- visit the [presidential finance map](http://127.0.0.1:8000/data/candidates/president/presidential-map/)
- confirm that 2012 and 2008 are options and that their data is correct
